packages/medusa/src/api/routes/admin/tax-rates/add-to-product-types.ts

```diff
@@ -37,6 +37,9 @@ import { validator } from "../../../../utils/validator"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostTaxRatesTaxRateProductTypesReq"
+ * x-codegen:
+ *   method: addProductTypes
+ *   queryParams: AdminPostTaxRatesTaxRateProductTypesParams
```

Comment:
Client was not referencing the correct queryParams type.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/tax-rates.ts#L100-L105